Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM openjdk:17-oraclelinux8 as builder
+
+USER root
+
+RUN jlink \
+    --module-path "$JAVA_HOME/jmods" \
+    --add-modules java.compiler,java.sql,java.naming,java.management,java.instrument,java.rmi,java.desktop,jdk.internal.vm.compiler.management,java.xml.crypto,java.scripting,java.security.jgss,jdk.httpserver,java.net.http,jdk.naming.dns,jdk.crypto.cryptoki,jdk.unsupported \
+    --verbose \
+    --strip-debug \
+    --compress 2 \
+    --no-header-files \
+    --no-man-pages \
+    --output /opt/jre-minimal
+
+USER app
+#
+## Now it is time for us to build our real image on top of an slim version of debian
+#
+FROM bitnami/minideb:bullseye
+COPY --from=builder /opt/jre-minimal /opt/jre-minimal
+
+ENV JAVA_HOME=/opt/jre-minimal
+ENV PATH="$PATH:$JAVA_HOME/bin"
+
+WORKDIR /home
+
+ARG jarFile
+ENV JARFILE=${jarFile}
+
+COPY $jarFile /home
+COPY *.yaml /home
+ENTRYPOINT java -jar ${JARFILE} -r resource-definition.yaml -s trace-definition.yaml -t cli-target-rest.yaml 
```

Comment:
Any dir paths need to be stripped off of the JARFILE env var.

Reply:
It wont be right? due to COPY command? if it is wrong COPY will fail?
Any specific use-cases?

Reply:
```
test-telemetry-generator % gw clean build jar fatJar -x test

test-telemetry-generator % docker build . -t test_ttg --build-arg jarFile=build/libs/test-telemetry-generator-all.jar 

test-telemetry-generator % docker run test_ttg
Error: Unable to access jarfile build/libs/test-telemetry-generator-all.jar
```